### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Upload_UploadFolder_ColorContrast:Color contrast ratio of "key" and "value" placeholder text is (1.519:1) which is less than required 4.5:1

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/AzureStorageExplorer#7582]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1 (96)

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

Windows

### OS Version

Windows 11 Enterprise Insider Preview (23H2)

### GitHub Tags:
#A11yMAS;#A11yTCS;#Win11;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_Win32_Dec2023;#Azure Tools Storage Explorer;#ColorContrast#WCAG1.4.3;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)

### Steps to Reproduce

1. Launch Storage Explorer
2. Tab till "Open Connect dialog" pane and Sign in to Azure.
3. Tab till "Manage Accounts" pane and Select the subscriptions you will use. Once selected, hit enter on "Apply" button.
4. Tab till "Blob Containers" of the selected subscription, press context menu button and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, tab till "Upload" button and hit enter on "Upload Folder" control.
6. Verify that Color contrast ratio of "key" and "value" placeholder text is greater than or equal to required 4.5:1 or not.

### Actual Experience

Color contrast ratio of "key" and "value" placeholder text is(1.519:1) which is less than required 4.5:1
**Note:** Similar issue is observed in Upload file and Get Shared Access Signature scenarios also.

### Expected Experience

Color contrast ratio of "key" and "value" placeholder text should be greater than or equal to 4.5:1.

### User Impact:
Low vision users will face difficulty in accessing the control if the color contrast ratio is less than (4.5:1) and placeholder text is not clearly visible to the user.

### Attachment:
![A11y_Azure Tools Storage Explorer_Upload_UploadFolder_ColorContrast](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/0729cfdb-175e-414c-ba59-7295bc8b6060)
![A11y_Azure Tools Storage Explorer_Get manage aceess signature_ColorContrast](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/24e1ce99-f810-457e-b52c-96ca072707e6)

**Comment by Rama8345** [comment]
Verified the issue at Microsoft Azure Storage Explore Version 1.33.0 (96) the issue is still repro'ing. Could you please provide latest build. Please find below attachment for reference.

![Screenshot 2024-01-16 151005](https://github.com/microsoft/AzureStorageExplorer/assets/115216656/b56d25da-5eab-4c58-8946-40919173621b)

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Microsoft Azure Storage Explorer
version: 1.33.0-dev(96)
Issue is fixed. hence closing the bug. Please find below added attachment for reference.
![7582_Fixed](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/fa8a3f07-dbe8-4afe-b58d-e281d47b2c67)
![7582_note isse fixed](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/4119ddc2-0d9b-46b2-88be-68386834ea00)
